queries/verilog/highlights.scm

```diff
@@ -267,8 +270,20 @@
 
 (struct_union) @type
```

Comment:
Can the identifier be captured?

Reply:
To make sure I understand, you are talking about identifier of the struct/union which should be `@variable` (unless  there is a `typedef`, in which case it should be `@type` as it is now)?

Reply:
Something like this:
````
(struct_union
  [
    "struct"
    "union"
  ] @keyword.type)
  (identifier) @type)
```
(and "enum" should not be captured as `@type` at all)

Reply:
Even with no `typedef`? I think the current reasoning is that without `typedef`, the identifier is a variable name itself, with type `enum`. With `typedef`, the identifier is a *type* name that other variables will be typed as

Reply:
That's not the case in C (as far as I can tell) so I don't see why it should be in verilog.